src/views/EditSidebar.vue

```diff
@@ -14,7 +13,23 @@
 		</template>
 
 		<AppSidebarTab name="Details" icon="icon-details" :order="0">
-			This is the details tab
+			<title-timepicker :event-component="eventComponent" />
```

Comment:
Title and time-picker should not be in the details tab, but inside primary actions, so they are always visible.